tests/fixtures/s3.py

```diff
@@ -36,14 +38,9 @@ def __init__(self, image: str, settings: Settings) -> None:
         self.ports = {'9000/tcp': 9100, '9101/tcp': 9101}
         # It's necessary to have 4 volumes to enable versioning (run MinIO in Erasure Code)
         # https://docs.min.io/docs/minio-erasure-code-quickstart-guide.html
-        self.volumes = ['test-data-1:/data1', 'test-data-2:/data2', 'test-data-3:/data3', 'test-data-4:/data4']
+        self.volumes = [f'test-data-{random.randint(100,999)}:/data{i}' for i in range(1, 5)]
         self._command = 'minio server /data{1...4} --console-address ":9101"'
 
-    def start(self) -> None:
-        for i in range(1, 4):
-            self._docker.client.volumes.create('test-data-' + str(i))
-        super().start()
-
     def stop(self, **kwargs) -> None:
         super().stop(**kwargs)
         self._docker.client.volumes.prune()
```

Comment:
Here how we can get volumes and remove them one by one:
```suggestion
        for volume_name in self.volumes.keys():
            try:
                self._docker.client.volumes.get(volume_name).remove()
            except DockerException:
                pass
```